app/page-partials/my-dashboard/TeamList.tsx

```diff
@@ -148,8 +148,8 @@ export default function TeamList({ currentUser, setTeam, loading, teams, loadTea
                   onClick={() => updateActiveTeam(team)}
                 >
                   <td>{team.name}</td>
-                  <td>
-                    <ActionButtonContainer>
+                  <td style={{ float: 'right', marginTop: '20px' }}>
+                    <ActionButtonContainer style={{ paddingRight: '4px' }}>
```

Comment:
i think here i need to use the `style attribute` for this specific place, since from the `ActionButtonContainer` definition, the default right padding is either 15px or 0 (which i used a `paddingRight property` to control)